docs/sftt/sftt-api-spec.md

```diff
@@ -252,23 +252,74 @@ No request body.
 
 ##### `200 OK`
 
-!!! missing "JSON body"
+```json
+{
+  "id": INT,
+  "name": STRING,
+  "bio": STRING,
+  "members": [
+    {
+      "user_id": INT,
+      "username": STRING,
+      "team_role": STRING,
```

Comment:
This is an enum right? We should specify what the valid team roles are somewhere in this file and then this type would be TEAM_ROLE instead of STRING